framework/src/task_manager/task_manager.c

```diff
@@ -231,15 +84,14 @@ int task_manager(int argc, char *argv[])
 	attr.mq_msgsize = sizeof(tm_request_t);
 	attr.mq_flags = 0;
 
-	builtin_cnt = get_builtin_list_cnt();
-	if (builtin_cnt < 2) {
+	if (get_builtin_list_cnt() < 1) {
 		tmdbg("Failed to start task manager\n");
```

Comment:
This debug output gives only failure as result. There is no reason. Let's change it as below:
```
No builtin data. Failed to start task manger.
```